src/systemcmds/pwm/pwm.c

```diff
@@ -649,18 +671,26 @@ pwm_main(int argc, char *argv[])
 						}
 					}
 
-					warnx("User abort\n");
+					PX4_INFO("User abort\n");
 					return 0;
 				}
 			}
 
-			usleep(2000);
+			/* Delay longer than the max Oneshot duration */
+
+			usleep(2542);
```

Comment:
> the actual timer value shall be rounded up to the next supported value

Ah, so the rounding makes sense to me. It means the usleep is fine and we're safe.
But why quantized up to 4ms? Is the resolution of usleep in NuttX only 4ms?